examples/minnietwitter/src/main/java/amino/run/appexamples/minnietwitter/MinnieTwitterMain.java

```diff
@@ -280,102 +281,99 @@ of detachFrom() just deletes the one and only reference to the microservice in O
      *
      * @param tm the TwitterManager to populate.
      */
-    private static void populate(TwitterManager tm) {
+    private static void populate(TwitterManager tm)
+            throws UnsupportedEncodingException, NoSuchAlgorithmException {
 
         List<Timeline> timelines = new ArrayList<Timeline>();
 
-        try {
-            /* Get User and Tag Manager */
-            UserManager userManager = tm.getUserManager();
-            TagManager tagManager = tm.getTagManager();
-            /*
-             * TODO quinton: The rest of this function is still really horrible code.
-             * It's the original UW code, that I've just moved here, but not had time to
-             * clean up properly yet.
-             */
-            /* Create the users */
-            List<User> users = new ArrayList<User>();
-            for (int i = 0; i < USER_COUNT; i++) {
+        /* Get User and Tag Manager */
+        UserManager userManager = tm.getUserManager();
+        TagManager tagManager = tm.getTagManager();
+        /*
+         * TODO quinton: The rest of this function is still really horrible code.
+         * It's the original UW code, that I've just moved here, but not had time to
+         * clean up properly yet.
+         */
+        /* Create the users */
+        List<User> users = new ArrayList<User>();
+        for (int i = 0; i < USER_COUNT; i++) {
+            long start = System.nanoTime();
+            User u = userManager.addUser("user" + i, "user" + i);
```

Comment:
Not related to this PR. we shall delete all the microservices created in runFullDemo() at the end of testcase. May be in separate PR.

Reply:
Agreed, will do in separate PR.